crates/astria-sequencer-relayer/src/metrics_init.rs

```diff
@@ -13,28 +13,47 @@ use metrics::{
 pub fn register() {
     celestia_client::metrics_init::register();
 
+    describe_counter!(
+        CELESTIA_SUBMISSION_COUNT,
+        Unit::Count,
+        "The number of calls made to submit to celestia"
+    );
+
     describe_counter!(
         CELESTIA_SUBMISSION_HEIGHT,
         Unit::Count,
         "The height of the last blob submitted to Celestia"
     );
 
     describe_counter!(
-        CELESTIA_SUBMISSION_COUNT,
+        CELESTIA_SUBMISSION_FAILURE_COUNT,
```

Comment:
The diff looks weird, but the point here is that `CELESTIA_SUBMISSION_COUNT` was described twice (instead of `CELESTIA_SUBMISSION_COUNT` and `CELESTIA_SUBMISSION_FAILURE_COUNT`)